.github/workflows/gradle.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+---
+name: Java CI with Gradle
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+      - trying
+      - "renovate/**"
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
```

Comment:
Im not sure about java 11 and druid. I think we should use 8 here. @fhennig what do you think?

Reply:
Ah, just saw this now. Yes the stuff needs to be compiled with source compatiblity 1.8, but it should work with the JDK 11 anyways

Reply:
Then its good to go! Maybe activate branch protection @teozkr?

Reply:
Can't activate the branch protection before at least one PR has been merged by Bors.

Reply:
Branch protection is now enabled.